### DISCUSSION TITLE
Locale inconsistencies

### DISCUSSION THREAD

**Discussion by wilsonadenuga** [gpbl/react-day-picker#2534] [Support] [answered]
## Code 

```tsx
import { DayPicker } from "react-day-picker";

export function Start() {
  return (
    <DayPicker locale={it}  />
  );
}
```

## Expected Behavior
I expect all aria-label and data attributes to be in Italian

## Actual Behavior
The aria-label for next and previous month button is still in English, also today is also in english

## Screenshots
<img width="570" alt="Screenshot 2024-10-10 at 11 23 09" src="https://github.com/user-attachments/assets/d7bd8fe1-d0dd-4579-8015-aadeaa667cba">

**Comment by gpbl** [comment, marked as answer]
@oluwatunmiisheii `aria-label`s are not included in the `locale`. To translate the labels, you will need to use the `labels` prop. 

See: https://daypicker.dev/docs/translation#example-italian-labels
